docs/src/user-guide/formatters/index.md

```diff
@@ -145,6 +183,10 @@ function addOne(i) {
 
 ### tap
 
+TODO: quick overview
```

Comment:
https://testanything.org/

When you want to pass the results to a system that expects this format.

Context: , https://github.com/eslint/eslint/issues/608